test/sampleData.js

```diff
@@ -0,0 +1,28 @@
+export const sampleData = 
```

Comment:
Great idea to nest all your sample data into one data object, and then call on each component of data by accessing the corresponding property. This limits the number of exports/imports you need.